core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/IntelliJFactoryTest.scala

```diff
@@ -0,0 +1,29 @@
+package org.virtuslab.ideprobe.dependencies
+
+import java.io.PrintWriter
+import java.nio.file.{Files, Path, Paths}
+
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Test
+import org.virtuslab.ideprobe.ide.intellij.IntelliJFactory
+
+import scala.io.Source
+
+@RunWith(classOf[JUnit4])
+final class IntelliJFactoryTest {
+
+
+  @Test
+  def test: Unit = {
+    val resourcePath = Paths.get(getClass.getResource(".").getPath)
```

Comment:
I'd suggest to put the product info in some subdirectory with meaningful name

Reply:
Will be done in the next commit.

Reply:
Done.